src/init.cpp

```diff
@@ -462,6 +462,8 @@ std::string HelpMessage(HelpMessageMode mode)
         strUsage += HelpMessageOpt("-maxsigcachesize=<n>", strprintf("Limit sum of signature cache and script execution cache sizes to <n> MiB (default: %u)", DEFAULT_MAX_SIG_CACHE_SIZE));
         strUsage += HelpMessageOpt("-maxtipage=<n>", strprintf("Maximum tip age in seconds to consider node in initial block download (default: %u)", DEFAULT_MAX_TIP_AGE));
     }
+    strUsage += HelpMessageOpt("-avoidpartialspends", strprintf(_("Group outputs by address, selecting all or none, instead of selecting on a per-output basis. Privacy is improved as an address is only used once (unless someone sends to it "
```

Comment:
"by key". Outputs don't have addresses.

Reply:
I see what you're saying, but in this case I think 'address' is fine; we speak of address reuse, after all. While outputs themselves don't have addresses, the funding to each output was done to a specific address; if that address was the same, then this PR will group them together.

Maybe `Group outputs by key (address), selecting all or...` would be fine, though?